README.md

```diff
@@ -28,13 +35,24 @@ cd chainweb-node
 nix-build
 ```
 
+This will place a runnable `chainweb-node` binary at a path like
+`/nix/store/d2f1ybvrlxj02ikfbgmjgfyybqbxk2s6-chainweb/ghc/chainweb/bin/chainweb-node`.
+This path changes with every code update, so a more reliable way to run
+`chainweb-node` is to use the symlink created by Nix:
+
+```bash
+./result/ghc/chainweb/bin/chainweb-node
+```
+
 ### Other Build Methods
 
-Requirements:
+#### Dependencies
 
-- Cabal >= 2.2
-- GHC >= 8.4
-- (optional) Stack >= 1.9
+- `ghc >= 8.4` (Haskell compiler) and `cabal >= 2.2` (Haskell build-tool)
+  - [Linux / Mac](https://www.haskell.org/ghcup/)
+  - [Windows](https://chocolatey.org/search?q=ghc) (via *chocolatey*)
+- (optional) `stack >= 1.9` (Alternate Haskell build-tool, which installs GHC for you)
```

Comment:
Not sure why this is "optional" when lead developers on our team rely on it.

Reply:
In terms of spending time supporting non-kadena devs, stack is strictly superior.

Reply:
I'm with you on this one.

Reply:
I don't have a strong opinion on wording, but stack is certainly not mandatory.